doc/source/getting_started/notebooks/ODL Basics.ipynb

```diff
@@ -0,0 +1,662 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ODL Basics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Welcome to your first tutorial. Here you will learn the basics of linear spaces, vectors and related functionality as used throughout ODL. The main purpose is to get you accustomed to working with the library, the \"look and feel\" of typical code and some building blocks and patterns that repeat themselves in other parts. We will explain them as we go along.\n",
+    "\n",
+    "At this point, you should have a working installation of ODL on your machine. If that's not the case, check out [the installation instructions](https://odlgroup.github.io/odl/getting_started/installing.html) and come back when you're finished!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import print_function  # We import this for Python 2 compatibility\n",
+    "import odl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Everything starts with importing the package. If you want to explore it, you can type `odl.` and hit `<TAB>` in an interactive shell (for example [IPython](https://ipython.org/)) to see all its top-level contents. The `odl` package is organized into several sub-packages, but the most important core functions and classes are available at the top level, so you don't have to type `odl.subpackage1.subpackage2.module.function` all the time.\n",
+    "\n",
+    "Now the simplest kind of mathematical object you can create in ODL is a *vector*. If you're familiar with [NumPy](http://www.numpy.org/), you can simply think of a `numpy.ndarray`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[1, 2, 3]\n"
+     ]
+    }
+   ],
+   "source": [
+    "x = odl.vector([1, 2, 3])\n",
+    "print(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In fact, the default implementation of a `vector` is just a thin wrapper around `numpy.ndarray`. However, this back-end can be switched for something else, e.g., a CUDA-based data container. On the surface it will look exactly the same, and the differences will be entirely under the hood. More on that later.\n",
```

Comment:
How to swap? At least mention it now that you have teased them, or make clickable.